tools/acrn-crashlog/README.rst

```diff
@@ -0,0 +1,124 @@
+ACRN-Crashlog
+#############
+
+Introduction
+************
+
+The ``ACRN-Crashlog`` is a debug tool to collect any useful logs/information after each crash/event occurred on the platform, including hypervisor, Service OS(SOS) and Android as a Guest(AaaG). It provides abundant logs for developers to debug/analyze the crashes. And it provides a flexible way to configure which crash/event to be concerned with xml file.
+
+Building
+********
+
+Build dependencies
+==================
+
+The ``ACRN-Crashlog`` tool depends on the following libraries (build and runtime):
+
+- libevent
+
+- OpenSSL
+
+- libxml2
+
+- systemd
+
+- telemetrics-client-dev (optional, detected at build time)
+
+For more detailed instructions on how to set-up your build environment, please refer to the :ref:`getting_started`.
+
+Build
+=====
+
+To build the ``ACRN-Crashlog``
+
+.. code-block:: console
+
+   make
+
+To Clean the build:
+
+.. code-block:: console
+
+   make clean
+
+Installing
+**********
+
+Install dependencies
+====================
+
+Since ``ACRN-Crashlog`` runs as a probe of `telemetrics-client`_, it needs `telemetrics-client`_ (>=1.6.0) to be available on the system.
```

Comment:
How do you get the telemetrics-client installed on the system?  And how do you check what version you have installed?

Also, don't make EVERY reference to telemetrics-client a link, particularly in the same sentence.

Reply:
> How do you get the telemetrics-client installed on the system? And how do you check what version you have installed?

It's installed on Clearlinux when you install the build dep (as per the GSG). For Ubuntu, Fedora et al. there is no *simple* instructions. The only way is to pull the source code/repo and build it locally. Having said that, the tool will compile fine if that package is not installed, it will just not use it either to report crash info when run.